modules/conversion-center/classes/render/render-base.php

```diff
@@ -166,78 +198,217 @@ protected function render_bio(): void {
+
+		// Set props for secondary image
+		if ( $output_images['secondary_image']['should_render'] ) {
+			$output_images['secondary_image']['props']['style'] = 'cover';
+			$output_images['secondary_image']['props']['show_bottom_border'] = $this->settings['identity_image_show_bottom_border'] ?? false;
+
+			// Handle shape divider
+			if ( ! empty( $this->settings['identity_section_style_cover_divider_bottom'] ) ) {
+				$output_images['secondary_image']['props']['has_shape_divider'] = true;
+
+				// Remove border if a shaped divider is applied
+				$output_images['secondary_image']['props']['show_bottom_border'] = false;
+			}
+
+			// Force primary to be profile since this is cover
+			$output_images['primary_image']['props']['style'] = 'profile';
 		}
-		$this->widget->add_render_attribute( 'identity_image', [
-			'class' => $identity_image_classnames,
-		] );
 		?>
 		<div class="e-link-in-bio__identity">
-			<div <?php echo $this->widget->get_render_attribute_string( 'identity_image' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
-				<?php if ( ! empty( $identity_image_value['id'] ) ) {
-					echo wp_get_attachment_image( $identity_image_value['id'], 'medium', false, [
-						'class' => 'e-link-in-bio__identity-image-el',
-					] );
-				} else {
-					$this->widget->add_render_attribute( 'identity_image_src', [
-						'alt' => '',
-						'class' => 'e-link-in-bio__identity-image-el',
-						'src' => esc_url( $identity_image_value['url'] ),
+			<?php
+			foreach ( $output_images as $image_key => $image ) :
+				if ( $image['should_render'] ) :
+					$image_classnames = 'e-link-in-bio__identity-image e-link-in-bio__identity-image-' . $image['props']['style'];
+					if ( ! empty( $image['props']['show_border'] ) || ! empty( $image['props']['show_bottom_border'] ) ) {
+						$image_classnames .= ' has-border';
+					}
+					if ( ! empty( $image['props']['shape'] ) && 'profile' === $image['props']['style'] ) {
+						$image_classnames .= ' has-style-' . $image['props']['shape'];
+					}
+					if ( ! empty( $image['props']['has_shape_divider'] ) ) {
+						$image_classnames .= ' has-shape-divider';
+					}
+					$this->widget->add_render_attribute( 'identity_image_' . $image_key, [
+						'class' => $image_classnames,
 					] );
 					?>
-					<img <?php echo $this->widget->get_render_attribute_string( 'identity_image_src' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?> />
-				<?php }; ?>
-			</div>
+						<div <?php echo $this->widget->get_render_attribute_string( 'identity_image_' . $image_key ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
+							<?php if ( ! empty( $image['value']['id'] ) ) {
+								echo wp_get_attachment_image( $image['value']['id'], 'medium', false, [
+									'class' => 'e-link-in-bio__identity-image-el',
+								] );
+							} else {
+								$this->widget->add_render_attribute( 'identity_image_src' . $image_key, [
+									'alt' => '',
```

Comment:
Wouldn't it be best practice to add an Alt text? We might run into problems with an empty alt attribute.

Reply:
@hein-obox Given the image is purely textural and provides no context to the content an empty alt attribute would normally be considered appropriate I believe. We also do not have a control to add this manually in the case of an image included from a URL.